src/Dockerfile.template

```diff
@@ -20,6 +20,16 @@ ENV PYTHON_VERSION={{ spec.version }} \
     LANG=en_US.UTF-8 \
     PIP_NO_CACHE_DIR=off
 
+{% if spec.el_version == '8' %}
+# RHEL7 base images atomatically set these envvars to run scl_enable. RHEl8
+# images, however, don't as most images don't need SCLs any more. But we want
+# to run it even on RHEL8, because we set the virtualenv environment as part of
+# that script
+ENV BASH_ENV=${APP_ROOT}/etc/scl_enable \	
+    ENV=${APP_ROOT}/etc/scl_enable \	
+    PROMPT_COMMAND=". ${APP_ROOT}/etc/scl_enable"
+{% endif %}
+
```

Comment:
nitpick: remove the line after `{% endif %}` so we do not have empty line additions in generated files?

Reply:
Yeah, I wanted to do it too. I tried doing `{% endif -%}` which removes the empty line from the generated files, but then in the RHEL8 files it's squished together, so I would have to add an empty line before the endif, and that just looks horrible. So I decided in favour of the beauty of the template instead of the beauty of the generated file. :)